Error while compiling MUF server with latest commit 9ee8bad5

@smaskell I am getting following compilation error with latest commit 9ee8bad is failing. error details:

/Users/sakoju/MUF-inMind-April2017-master/src/main/java/edu/cmu/inmind/multiuser/sara/component/SocialReasonerComponent.java:74: error: cannot find symbol
            socialController.getSocialReasoner().getNetwork().updateState(model.getBehaviorNetworkStates());
                                                ^
  symbol:   method getNetwork()
  location: class SocialReasoner
/Users/sakoju/MUF-inMind-April2017-master/src/main/java/edu/cmu/inmind/multiuser/sara/component/SocialReasonerComponent.java:101: error: cannot find symbol
            srOutput.setStates(socialController.getSocialReasoner().getNetwork().getState());
                                                                   ^
  symbol:   method getNetwork()
  location: class SocialReasoner

--- a/src/main/java/edu/cmu/inmind/multiuser/socialreasoner/control/reasoners/SocialReasoner.java
+++ b/src/main/java/edu/cmu/inmind/multiuser/socialreasoner/control/reasoners/SocialReasoner.java
@@ -139,4 +139,8 @@ public class SocialReasoner {
         instance.network.resetAll();
         instance = null;
     }
+
+    public BehaviorNetwork getNetwork() {
+        return network;
+    }
 }

